public/static/docs/command-reference/update.md

```diff
@@ -27,20 +27,32 @@ Note that import stages are considered always locked, meaning that if you run
 update them.
 
 `dvc update` will not have an effect on import stages that are fixed to a commit
-hash (`rev` field in the DVC-file). Please refer to
-[Fixed revisions & re-importing](/doc/command-reference/import#example-fixed-revisions-re-importing)
-for more details.
+hash (`rev` field in the DVC-file). To update the imported artifacts to a
+certain revision, `--rev` with specified revision can be used.
+
+```dvc
+dvc update --rev master
+```
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the repository to update
+  the file or directory from (also starts tracking the given revision).
+
+  > Note that this adds or updates a `rev` field in the DVC-file that fixes it
+  > to this revision (and updates `rev_lock` in the DVC-file). This can have an
+  > impact on the behavior of `dvc update` later.
```

Comment:
Lets just copy this from import:

```suggestion
- `--rev` - commit hash, branch or tag name, etc. (any
  [Git revision](https://git-scm.com/docs/revisions)) of the repository to
  update the file or directory from. The latest commit in `master` (tip of the
  default branch) is used by default when this option is not specified.

  > Note that this adds a `rev` field in the import stage that fixes it to this
  > revision. This can impact the behavior of `dvc update`. (See
  > **re-importing** example below.)
```